frontend/frontend-filters-links.php

```diff
@@ -533,6 +534,15 @@ protected function get_queried_term_id( $tax_query ) {
 			'fields' => 'ids',
 		);
 		$terms = get_terms( $args );
+
+		$tr_term = $this->model->term->get_translation( reset( $terms ), $lang );
+
+		if ( $tr_term ) {
```

Comment:
```suggestion
		if ( $tr_term && in_array( $tr_term, $terms ) ) {
```
You must check that the translated term has the same slug.